### ISSUE TITLE
[BUG] Incorrect scaling when multiple monitors have different scale

### ISSUE THREAD

**Issue by nemchik** [marticliment/WingetUI#1555]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have tested that this issue has not been fixed in the latest [(beta or stable) release](https://github.com/marticliment/WingetUI/releases/).
- [X] I have checked the [FAQ](https://github.com/marticliment/WingetUI#frequently-asked-questions) section for solutions
- [X] This issue is about a bug (if it is not, please use the [correct template](https://github.com/marticliment/WingetUI/issues/new/choose))

### Describe your issue

WingetUI appears fine on my secondary screen (100% scale) but is scaled incorrectly (too small) when I open it or move the window to my primary screen (125% scale). If I disconnect my external screen WingetUI scales correctly to 125% on my laptop screen, but as soon as I reconnect my external screen WingetUI re-scales itself to 100% even if the window remains on my laptop screen, which should be scaled to 125%

### WingetUI Log

```text
not relevant (nothing shows up in the logs when moving the window from one monitor to the other). if needed I can post these as a reply.
```

### Package Managers Logs

```text
not relevant (nothing shows up in the logs when moving the window from one monitor to the other). if needed I can post these as a reply.
```

### Screenshots and videos

Correctly scaled on external monitor (right) at 100%
![image](https://github.com/marticliment/WingetUI/assets/725456/2e46d728-caa7-4967-8dec-5e52bf54879a)
Incorrectly scaled on internal (laptop) screen (left) at 125%
![image](https://github.com/marticliment/WingetUI/assets/725456/b2ad03f9-bf03-45bc-960c-b6b0a4c814bb)
Correctly scaled when only internal (laptop) screen is active at 125%
![image](https://github.com/marticliment/WingetUI/assets/725456/60b04f22-64a3-4a7f-861b-7311be01b954)
I know it's not easy to tell from the screenshots without opening them up and zooming to scale, but if you do you should be able to tell the difference.

**Comment by marticliment** [comment]
This issue should have been fixed on Qt 6.6.0. I am going to close it by now since next WingetUI version will have Qt 6.6.0

**Comment by marticliment** [comment]
If this issue persists on next version please let me know
